### PR TITLE
Add package nimhdf5

### PR DESCRIPTION
**Pull request by Vindaar**
Oops, nimble used the wrong URL. I'll fix that later today, sorry about that.

### CODE REVIEW

**Comment by Vindaar** [comment]
Ok, fixed everything.

File: packages.json [hunk 1]
```diff
@@ -8745,5 +8745,16 @@
     "description": "Nim bindings for Clutter toolkit.",
     "license": "LGPLv2.1",
     "web": "https://github.com/KeepCoolWithCoolidge/nimclutter"
+  },
+  {
+    "name": "nimhdf5",
+    "url": "https://github.com/Vindaar/nimhdf5",
+    "method": "git",
+    "tags": [
+      "v0.2.7"
```
Comment (dom96) [inline packages.json]: tags are not for versions

Comment (dom96) [reply packages.json]: Yeah, sorry. Nimble should be clearer here :)

**Comment by Vindaar** [comment]
Added proper tags.

**Comment by dom96** [comment]
Thanks :)

**Pull request merged**